bindings/CMakeLists.txt

```diff
@@ -5,6 +5,10 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 if(pyroot)
+  if (NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR AND ((how_many_pythons GREATER 1) OR MSVC OR pyroot_legacy))
```

Comment:
> if (NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR

This will probably break Windows, where `CMAKE_INSTALL_PYTHONDIR` is by default `CMAKE_INSTALL_BINDIR`, not `CMAKE_INSTALL_LIBDIR`.

Reply:
thanks, i'll update the condition!

Reply:
condition updated, it's now `if(MSVC OR (NOT CMAKE_INSTALL_LIBDIR STREQUAL CMAKE_INSTALL_PYTHONDIR AND how_many_pythons GREATER 1)`